sdk/src/main/java/com/horizen/account/api/rpc/types/EthereumBlockView.java

```diff
@@ -64,10 +64,10 @@ public EthereumBlockView(Long blockNumber, String hash, boolean hydratedTx, Acco
         this.timestamp = Numeric.prependHexPrefix(Long.toHexString(block.timestamp()));
         this.baseFeePerGas = Numeric.toHexStringWithPrefix(blockHeader.baseFee());
 
-        var transactions = (List<EthereumTransaction>) scala.collection.JavaConverters.seqAsJavaList((block.transactions()));
+        var transactions = scala.collection.JavaConverters.seqAsJavaList(block.ethereumTransactions());
```

Comment:
Ok, i got the issue on java side.
But it is not good to keep `ethereumTransactions` method inside the AccountBlock. No one should use it.
Let's instead move it to some scala util object in the `rpc` package.